src/transactions/dto/transactions.dto.ts

```diff
@@ -0,0 +1,25 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { TransactionType } from "@prisma/client";
+
+export class Transaction {
```

Comment:
@IsInt for number types, non-negativity constraint for credits.

Reply:
This dto is for response actually